Standards/scs-0104-w1-standard-images-implementation.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "SCS Standard Images Standard: Implementation Notes"
+type: Supplement
+track: IaaS
+status: Proposal
+supplements:
+  - scs-0104-v1-standard-images.md
+---
+
+## Introduction
+
+The SCS standard on standard images does not in itself lay down what images are actually
+required or recommended; rather it specifies the format of a YAML file that in turn serves
+said purpose. The particular YAML file that an implementer (a cloud service provider or operator)
+has to comply with is given in the respective version of the certificate scope "SCS-compatible IaaS"
+as a parameter to the standard. This document is intended to give implementers a
+step-by-step guide on how to comply with the SCS certificate scope.
+
+## Step-by-step walkthrough
+
+### Option A: pragmatic
+
+Run the test script on your environment and check the error messages :)
+
+1. Check out the [standards repository](https://github.com/SovereignCloudStack/standards).
+
+   ```shell
+   git clone https://github.com/SovereignCloudStack/standards.git
+   cd standards
+   ```
+
+2. Install requirements:
+
+   ```shell
+   python3 -m venv .venv && source .venv/bin/activate
+   pip install -r requirements.txt 
+   ```
+
+3. Make sure that your `OS_CLOUD` environment variable is set.
+4. Run the main check script:
+
+   ```shell
+   python3 ./Tests/scs-compliance-check.py ./Tests/scs-compatible-iaas.yaml -s $OS_CLOUD -a os_cloud=$OS_CLOUD -o report.yaml -C
+   ```
```

Comment:
I would add `-t standard-images-check` once we have the PR https://github.com/SovereignCloudStack/standards/pull/634. Please review.